content/workshops/openshift33/Setup.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Workshop Prerequisites
+workshops: openshift33
+workshop_weight: 10
+layout: lab
+---
+
+### If we provided you a laptop or cloud environment
```

Comment:
This must have been a deliberate style choice of the original content creators.  I retained the level 3 headings during conversion.  This will be reset from level 3 to level 1 headings as requested.